doc/rst/language/spec/statements.rst

```diff
@@ -699,6 +700,65 @@ A ``break`` statement cannot be used to exit a parallel loop
         }
       }
 
+.. _The_Require_statement:
+
+The Require Statement
+---------------------
+
+The require statement provides a means to specify required files from
+within the program. It has an effect similar to adding the specified
+files to the Chapel compiler's command line. The filenames are relative
+to the directory from which the Chapel compiler was invoked. Any
+directories specified using the Chapel compiler's -I or -L flags will
+also be searched for matching files.
+
+.. code-block:: syntax
+
+   require-statement:
+     'require' string-or-identifier-list
+
+   string-or-identifier-list:
+     string-or-identifier
+     string-or-identifier ',' string-or-identifier-list
+
+   string-or-identifier:
+     string-literal
+     identifier
+
+The require statement takes a comma-separated list of param string
+filenames as operands. The filenames must be Chapel source files (*.chpl),
```

Comment:
I suggest to leave the "param string" part for the next paragraph, see below, as the current phrasing was not clear to me. Also switch to using singular in the second sentence. For example:

The require keyword must be followed by a list of filenames. Each filename must be a Chapel source file (*.chpl), a C source file [and so on].